hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1746,7 +1746,7 @@ public boolean balance(boolean force) throws IOException {
           toPrint = regionsInTransition.subList(0, max);
           truncated = true;
         }
-        LOG.info(prefix + "unning balancer because " + regionsInTransition.size() +
+        LOG.info(prefix + "Running balancer because " + regionsInTransition.size() +
```

Comment:
Ideally, this change is not required since prefix takes care of it. 
However, to improve text searching, we can make this change:
```
String prefix = force && !metaInTransition ? "" : "Not ";
```

Reply:
+1 to @virajjasani  comment.

Reply:
Ouch, haven't realized that prefix. Will revert this piece. I believe it exists this way for appropriate capitalization, so don't want to break it.